### PR TITLE
website: Remove Netlify Edge stuff

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
This change shaves off about 2.2 seconds for the initial boot for the app. Reloads are incremental, and thus shouldn't be impacted.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,15 +7,15 @@
     "license": "Apache-2.0",
     "author": "Nick O'Leary",
     "scripts": {
-        "start": "npm-run-all clean:dev build:js --parallel dev:*",
```
Comment (joepavitt) [inline package.json:10]: why remove `clean:dev`?

File: package.json [hunk 2]
```diff
@@ -7,15 +7,15 @@
...
+        "start": "npm-run-all build:js --parallel dev:*",
```
Comment (joepavitt) [inline package.json:10]:
```suggestion
        "start": "npm-run-all clean:dev build:js --parallel dev:*",
```

Comment (joepavitt) [reply package.json:10]: Revert this, and I'm happy with this PR

**Comment by ZJvandeWeg** [comment]
Should be good to go @joepavitt

**Review by joepavitt — APPROVED** [review]

**Pull request merged**